theforeman.org/yaml/jobs/test_plugin_pipeline.yaml

```diff
@@ -0,0 +1,135 @@
+# parameters:
+#   name: plugin name, e.g. foreman_bootdisk
+#   repo: git repo name as it may differ, e.g. foreman_bootdisk
+#   branch: git repo branch to test, e.g. master
+#   foreman_branch: foreman git repo branch to run on, e.g. develop
+- job-template:
+    name: 'test_foreman_plugin_{plugin}_{branch}'
+    project-type: pipeline
+    sandbox: true
+    properties:
+      - github:
+          url: '{repo}'
+      - tfm-build-discarder
+    parameters:
+      - string:
+          name: plugin_repo
+          default: '{repo}'
+          description: "Git URL containing the Foreman plugin, e.g. <pre>https://github.com/theforeman/foreman_example</pre>"
+      - string:
+          name: plugin_branch
+          default: '{branch}'
+          description: "Git branch name of the Foreman plugin, e.g. <pre>master</pre>"
+      - string:
+          name: plugin_name
```

Comment:
It's odd to me to generate a dedicated job to a plugin and then be able to change the name of the plugin as an input to the job.